src/Change.php

```diff
@@ -77,6 +77,7 @@ public function withFilePositionsIfNotAlreadySet(
         int|null $line,
         int|null $column,
     ): self {
+        /** @infection-ignore-all */
```

Comment:
Should not ignore here: need to dig down to why infection adds mutations here :thinking:

Reply:
This one is because it can remove the `clone` keyword. I wasn't sure what was the intention to do that.

Reply:
This one is because it can remove the `clone` keyword. I wasn't sure what was the intention to use it.

Reply:
This one is because it can remove the clone keyword.

Reply:
@michael-rubel yeah, this kind of signaling (infection removing stuff) indicates that we have a test to add, BTW :)

Reply:
Yeah, I was unsure if it makes sense to cover.